### PR TITLE
feat(): add tenant backup destType

### PR DESCRIPTION
**Pull request by TianWuwt**
## Summary

add tenant backup destType( COS、S3、 S3-Compatible)

## Solution Description

image
image
image

### CODE REVIEW

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx [hunk 1]
```diff
@@ -250,6 +253,61 @@ export default function NewBackup() {
                     />
                   </Form.Item>
                 </Col>
+
```
Comment (powerfooI) [inline ui/src/pages/Tenant/Detail/NewBackup/index.tsx:256]: Input form field `Host` should be added to the form if `destType` is not `NFS`. Refer to PRD to see the details.

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx [hunk 2]
```diff
@@ -29,8 +29,11 @@ export default function NewBackup() {
   const scheduleValue = Form.useWatch(['scheduleDates'], form);
 
   const distType = [
```
Comment (powerfooI) [inline ui/src/pages/Tenant/Detail/NewBackup/index.tsx:31]: This seems to be a typo. Maybe `destTypes` is better.

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx [hunk 3]
```diff
@@ -250,6 +253,61 @@ export default function NewBackup() {
...
+                            label="AppID"
+                            name={['appID']}
+                            rules={[
+                              {
+                                required: true,
+                                message: '请输入AppID',
+                              },
+                            ]}
+                          >
+                            <Password
```
Comment (powerfooI) [inline ui/src/pages/Tenant/Detail/NewBackup/index.tsx:272]: Plain text is more proper.

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx (hunk 1 above)
Comment (TianWuwt) [reply ui/src/pages/Tenant/Detail/NewBackup/index.tsx:256]: It was originally there
image

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx (hunk 3 above)
Comment (TianWuwt) [reply ui/src/pages/Tenant/Detail/NewBackup/index.tsx]: DONE

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx (hunk 1 above)
Comment (powerfooI) [reply ui/src/pages/Tenant/Detail/NewBackup/index.tsx:256]: I mean that a new field named `Host` should be added to the form when selecting COS, OSS, S3 or S3-Compatible.

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx (hunk 2 above)
Comment (TianWuwt) [reply ui/src/pages/Tenant/Detail/NewBackup/index.tsx]: done

File: ui/src/pages/Tenant/Detail/NewBackup/index.tsx (hunk 1 above)
Comment (TianWuwt) [reply ui/src/pages/Tenant/Detail/NewBackup/index.tsx:275]: done

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**